src/bin/pgcopydb/pgsql.c

```diff
@@ -2745,6 +2748,8 @@ pg_copy_data(PGSQL *src, PGSQL *dst, CopyArgs *args)
 	bool failedOnSrc = false;
 	bool failedOnDst = false;
 	args->bytesTransmitted = 0;
+	args->bytesTransmittedBeforeSavingProgress = 0;
+	args->lastSavingTimeMs = time(NULL);
```

Comment:
That looks like it should be handled in the context for the CopyProgressCallback function.